iphone/Classes/TiUILabel.m

```diff
@@ -420,6 +420,14 @@ -(void)setVerticalAlign_:(id)value
         [self padLabel];
     }
 }
+
+-(void)setMaxLines_:(id)value
+{
+    [[self label] setNumberOfLines:[TiUtils floatValue:value]];
+    [self padLabel];
+	[(TiViewProxy *)[self proxy] contentsWillChange];
```

Comment:
The indentation is messed-up here, please align with the other lines (use diff-tool to check before commiting).